pinot-core/src/test/java/org/apache/pinot/core/query/optimizer/filter/TimePredicateFilterOptimizerTest.java

```diff
@@ -50,15 +50,18 @@ public void testTimeConvert() {
         new Range(1620777600000L, true, 1620864000000L, false));
 
     // Other input format
-    testTimeConvert("timeConvert(col, 'MINUTES', 'SECONDS') > 1620830760", new Range(27013846L, false, null, false));
-    testTimeConvert("timeConvert(col, 'HOURS', 'MINUTES') < 27015286", new Range(null, false, 450254L, true));
+    testTimeConvert("timeConvert(col, 'MINUTES', 'SECONDS') > 1620830760",
```

Comment:
(format) There are some wrong auto-format. Can you double check your formatter setting?